testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py

```diff
@@ -94,9 +99,21 @@ def test_delta_sync_replication(params_from_base_test_setup, num_of_docs, replic
         cbl_db_docs = db.getDocuments(cbl_db, doc_ids)
         for doc_id, doc_body in cbl_db_docs.items():
             for i in range(number_of_updates):
-                doc_body["new_field_1"] = random_string(length=30)
-                doc_body["new_field_2"] = random_string(length=80)
-                db.updateDocument(database=cbl_db, data=doc_body, doc_id=doc_id, attachments_name=file_attachment)
+                if file_attachment:
+                    mutable_dictionary = dictionary.toMutableDictionary(doc_body)
+                    dictionary.setString(mutable_dictionary, "new_field_1", random_string(length=30))
+                    dictionary.setString(mutable_dictionary, "new_field_2", random_string(length=80))
+
+                    if liteserv_platform == "android":
+                        blob_value = blob.create("image/jpeg", blob.createImageContent("/assets/golden_gate_large.jpg"))
+                    elif liteserv_platform == "xamarin-android":
```

Comment:
may use file attachment parameter name to construct this, instead of manually have the file name "golden_gate_large.jpg"

Reply:
ditto

Reply:
There is a reason for not using it. These golden_gate image should store it in TestServer app, so that when it loads the app, it can find. It is kind of hard coded as only one image exists in app. If you parameterized and later if you change the parameter to some other image, this will fail saying image not found .